packages/terra-responsive-element/src/ResponsiveElement.scss

```diff
@@ -0,0 +1,3 @@
+.terra-ResponsiveElement {
+  width: 100%;
```

Comment:
This was added as a promise / safeguard, mainly for issues we have seen in the past with flexbox. I went ahead and removed the class and CSS. It can be added back if it ever becomes necessary.